xpressUI.css

```diff
@@ -98,3 +105,72 @@
     height: 50px;
     border-radius: 30%;
   }
+
+
+  /*-------------------Alert-----------------*/
+
+  .alert-component{
+  display: flex;
+  justify-content: center;
+  font-weight: bold;
+  }
+
+
+  .done{
+color: var(--success);
+  }
+
+  .warning{
+    color: var(--warning);
+  }
+
+  .error{
+    color: var(--error);
+  }
```

Comment:
try to use more descriptive classnames, here, its not very clear from the name what they are trying to do. Instead, you can name them like, `.txt-warning | error | success etc.` since, they are changing the text color only